# A synchronized pair of RGB and depth images from an RGB-D camera.
string camera_name
sensor_msgs/CameraInfo camera_info
sensor_msgs/Image rgb
sensor_msgs/Image depth
